src/v/cluster/types.h

```diff
@@ -613,12 +613,38 @@ struct create_data_policy_cmd_data {
 };
 
 struct non_replicable_topic {
-    static constexpr int8_t current_version = 1;
+    static constexpr int8_t current_version = 66;
```

Comment:
is it random number or it makes sense?

Reply:
Oops, good catch, that was comitted accidentally.